File: Java/Tarea 2.6/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio3 {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        Float num1;
        Float num2;
        Float suma;
        Float resta;
        Float div;
        Float mult;

        System.out.println("Introduce un primer numero: ");
        num1 = teclado.nextFloat();

        System.out.println("Introduce un segundo numero: ");
        num2 = teclado.nextFloat();

        System.out.println("---------------------------");
        suma = num1 + num2;
        System.out.println(num1 + " + " + num2 + " = " + suma);
        resta = num1 - num2;
        System.out.println(num1 + " - " + num2 + " = " + resta);
        mult = num1 * num2;
        System.out.println(num1 + " * " + num2 + " = " + mult);
        div = num1 / num2;
        System.out.println(num1 + " / " + num2 + " = " + div);
        
        teclado.close();
    }
}

File: Python/Examen practico/JH-C2e1Proj/ejercicio3.py
num1 = int(input("Introduce el primer numero: "))
num2 = int(input("Introduce el segundo numero: "))

for j in range(num1, num2 + 1):
    print(f"La tabla de multiplicar del {j}, desde el 10 hasta el 1, es:")
    print("----------------------------------------------------------")
    for i in range(10, 0, -1):
        total = i * j
        print(f"{j} x {i} = {total}")